src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,56 @@
+import cn from 'classnames';
+import { FC, useEffect, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+};
+
+export const Tabs: FC<Props> = ({ tabs }) => {
+  const [currentTab, setCurrentTab] = useState<Tab | null>();
+  const { tabId = null } = useParams();
+
+  const findTab = () => {
+    const tab = tabs.find(t => t.id === tabId);
+
+    if (tab) {
+      setCurrentTab(tab);
+    } else {
+      setCurrentTab(null);
+    }
+  };
+
+  useEffect(() => {
+    findTab();
+  }, [tabId]);
+
```

Comment:
Or you can create just one variable)

Reply:
one variable with useMemo sounds better here